File: Exercise01/C/problem1.c
#include<stdio.h>
int main(){
	float total, hours, hourly;
	setbuf(stdout, NULL);
	printf ("Enter the hourly pay rate and the number of hours worked separated by a space.\n");
 	scanf ("%f %f", &hourly, &hours);
    	total = hours*hourly;
      	printf ("Your total pay is: $%.2f", total);
}

File: Exercise01/Fortran/problem1.f
	program pay
		real :: total, hourly, hours
		print*, "Please enter the hourly pay rate"
		read(*,*) hourly
		print*, "Please enter the number of hours worked"
		read(*,*) hours
		total = hourly* hours
		print*, "Your total pay is $", total
	end program pay
